### DISCUSSION TITLE
Decentralised identifiers

### DISCUSSION THREAD

**Discussion by jainneerajnj** [INCOMS/vc-specs#8] [General]
•	POV on decentralised identifiers , how to create, register them and build for interoperability
•	POV on how to map the decentralised identifiers with the Aadhaar ID or Functional IDs depending on the sectors
•	Are there any open APIs defined for issuer registries to ensure interoperability?

**Comment by surendrasinghs** [comment]
@dileepbapat @tejash-jl @pramodkvarma @coolbung @parthlawate @code-blooded-guy tagging you if you missed the discussion on Discord.

**Reply by pramodkvarma** [reply]
Do share Discord invite link here @surendrasinghs 

Others, we have moved all discussions from slack to discord.

**Reply by surendrasinghs** [reply]
@jainneerajnj Please join https://discord.gg/muHqfEkvSy and bring others along :)
